lockbox-iosTests/ItemDetailActionSpec.swift

```diff
@@ -0,0 +1,56 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Quick
+import Nimble
+import RxSwift
+
+@testable import Lockbox
+
+class ItemDetailActionHandlerSpec : QuickSpec {
```

Comment:
Nit: I've been biting my tongue on your whitespace around colons. Choose a convention, stick to it. 😛

Reply:
Swiftlint PR just merged, all this will get fixed when I rebase.

Reply:
👍